packages/integration-sdk-core/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@jupiterone/integration-sdk-core",
+  "version": "0.1.0",
+  "description": "The SDK for developing JupiterOne integrations",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "repository": "git@github.com:JupiterOne/integration-sdk.git",
```

Comment:
This is stale now, may be worth addressing as these files are created.

Reply:
Good catch, changing now